packages/docs/components/link.md

```diff
@@ -21,10 +20,6 @@ Links are navigation elements displayed as text. A link can open another page or
 
 The <code>:active</code> state of a link does not have any unique styling, so it matches the <code>:hover</code> styling.
 
-### Anchor
```

Comment:
Why remove this section? It still applies, no?

Reply:
Pulling it out until we have actual guidance on how to use + examples. Currently, we don't do anything special for deep links.

Even the title here is a little misleading.